src/main/java/uk/gov/pay/api/model/directdebit/mandates/GetMandateResponse.java

```diff
@@ -35,7 +35,7 @@ private GetMandateResponse(String agreementId,
     public static GetMandateResponse from(MandateConnectorResponse mandate, MandateLinks links) {
         return new GetMandateResponse(
                 mandate.getMandateId(),
-                null,
+                mandate.getProviderId(),
```

Comment:
Can a model be shared for the create mandate and get mandate response?

Reply:
They can. I'll do a new PR for it.